qa/src/test/java/io/camunda/migrator/qa/SubprocessMigrationTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+package io.camunda.migrator.qa;
+
+import static io.camunda.migrator.qa.MigrationTestConstants.LEGACY_ID_VAR_NAME;
+import static io.camunda.process.test.api.CamundaAssert.assertThat;
+import static io.camunda.process.test.api.assertions.ElementSelectors.byId;
+import static io.camunda.process.test.api.assertions.ProcessInstanceSelectors.byProcessId;
+import static io.camunda.process.test.api.assertions.UserTaskSelectors.byTaskName;
+
+import java.util.stream.Stream;
+import org.camunda.bpm.engine.runtime.ProcessInstance;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.provider.Arguments;
+
+public class SubprocessMigrationTest extends RuntimeMigrationAbstractTest{
+
+  @Test
+  public void migrateCallActivityAndSubprocess() {
+    // given
+    deployProcessInC7AndC8("calledProcessInstance.bpmn");
+    deployProcessInC7AndC8("callActivity.bpmn");
+    ProcessInstance instance = runtimeService.startProcessInstanceByKey("callingProcessId");
+    ProcessInstance calledInstance = runtimeService
+        .createProcessInstanceQuery()
+        .superProcessInstanceId(instance.getProcessInstanceId())
+        .singleResult();
+    // when
+    runtimeMigrator.migrate();
+
+    // then
+    assertThat(byProcessId("callingProcessId")).isActive()
+        .hasActiveElements(byId("callActivityId"))
+        .hasVariable("legacyId", instance.getProcessInstanceId());
+    assertThat(byProcessId("calledProcessInstanceId")).isActive()
```

Comment:
Question: there is a `rootProcessInstanceId` field on `ProcessInstance`, would that field have the parent Id for child process instance here? If so, can we also add an assert on that

Reply:
Ah interesting. So this will not be set during migration then? Is this at all relevant for execution purposes?

Reply:
Given that the subprocess is created by C8 when the parent process call activity is activated we don't need to set it.